src/views/Verbose.vue

```diff
@@ -71,6 +71,16 @@
             v-model="filter"
             :placeholder="$t('verbose.filterPlaceholder')"
           >
+          <div
+            v-for="value in ['true', 'false', 'undefined']" :key="value"
+            :class="['exclude-result', { selected: isExcluded(value) }]"
+            @click="excludeResult(value)"
+          >
+            <span></span>
+            <p>
+              {{ $t('verbose.exclude') }} <code :class="value">{{ value }}</code>
```

Comment:
Please add language definitions for `verbose.exclude`
<img width="256" alt="image" src="https://user-images.githubusercontent.com/10846855/176338716-93face64-77e1-4df3-a878-68b19e6b8a0d.png">